get-clients/readme.md

```diff
@@ -0,0 +1,141 @@
+> Goal of this guide: to help you with direct customer acquisition for New Atoms services
+
```

Comment:
You should say here what you suggested in our conversation: that there are no hard and fast truths.